src/lib/mongoBackend/MongoCommonSubscription.cpp

```diff
@@ -202,33 +202,43 @@ void setEntities(const Subscription& sub, BSONObjBuilder* b)
   BSONArrayBuilder entities;
   for (unsigned int ix = 0; ix < sub.subject.entities.size(); ++ix)
   {
-    EntID en = sub.subject.entities[ix];
+    EntID       en        = sub.subject.entities[ix];
+    std::string finalId;
+    std::string finalType;
+    std::string isIdPattern;
+    bool        isTypePattern = false;
```

Comment:
This here is a little funny ...
isTypePattern is bool, but isIdPattern is a string?
If this is changable, bool is the better choice.